pages/airdrops/create.tsx

```diff
@@ -296,7 +294,7 @@ const CreateAirdrop: NextPage = () => {
           </label>
           <input
```

Comment:
should these inputs be turned into a reusable component?

Reply:
I haven't updated the create page, will push and request review when it's ready. 👍🏻